### ISSUE TITLE
No dejar tener más de un onTick con el mismo nombre

### ISSUE THREAD

**Issue by PalumboN** [uqbar-project/wollok#1854]
Esto es para que no meter bugs al hacer remove... Y no tirar _onTick_s a mansalva.

**Comment by lspigariol** [comment]
Porque sucede muchas veces es que el ontick esta dentro de un foreach, o en
un método que se llama muchas veces. (Por ejemplo cada proyect suelen tener
un ontick que lo hace mover hasta que colisiona)
No será mejor tener una clase Tick y métodos addTick y removeTick en game?

**Comment by PalumboN** [comment]
Sí, por eso quería discutirlo. Para esos casos yo les recomiendo usar `self.identity()` dentro del nombre del tick, pero no me agrada mucho.
Podríamos crear la clase `Tick` y que los juegos se encarguen de manejar la lista. Si hacemos eso prefiero que directamente los objetos entiendan los mensajes `start()` y `stop()`.

**Comment by PalumboN** [comment]
De la reunión de Wollok salió esta propuesta:

Reificar el concepto de tick para permitir referenciar a cada tick

Tener un método o poder instanciarlo desde la clase

const tick = game.tick(4000, { self.disparar() }) // immediate = true
const tick = new Tick(interval = 4000, action={ self.disparar() }, immediate = true)

**Cosas importantes***
// El tick va a intentar evaluar el bloque cada n milisegundos, si no puede en ese tiempo se ejecutará tantas veces como era esperado originalmente
// immediate dice si se ejecuta en el momento 0 o espera para la priemera ejecución.

Ejemplo de uso:
```Wollok
tick.start()
tick.interval(1000) // Reinicia contador
tick.action({ … })
tick.stop()
```
